### PR TITLE
ISPN-14566 Implement RESP access logging

### PR DESCRIPTION
**Pull request by jabolina**
https://issues.redhat.com/browse/ISPN-14566

I wanted to get this one out of the way already.

Running with memtier to compare the main and this branch with the access log disabled, the performance holds. A small snippet from the log:

```text
127.0.0.1 admin [25/Aug/2023:11:49:08 -0300] "GET /[[B0x6D656D746965722D393939] RESP" OK 27 9009 0
127.0.0.1 admin [25/Aug/2023:11:49:08 -0300] "SET /[[B0x6D656D746965722D343235] RESP" OK 9030 5 0
127.0.0.1 admin [25/Aug/2023:11:49:08 -0300] "GET /[[B0x6D656D746965722D343135] RESP" OK 27 5 0
```

GET with a hit, SET, and GET with a miss. Memtier is running with a 9K data size.

I think the time interval at the end is small and goes to 0.

### CODE REVIEW

File: server/resp/src/main/java/org/infinispan/server/resp/logging/Tracker.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+import java.time.temporal.Temporal;
+import java.util.List;
+
+import org.infinispan.commons.time.TimeService;
+import org.infinispan.server.resp.RespCommand;
+
+import io.netty.channel.ChannelHandlerContext;
+
+public class Tracker {
```
Comment (karesti) [inline server/resp/src/main/java/org/infinispan/server/resp/logging/Tracker.java:11]: some docs will be helpful here too

**Review by karesti — APPROVED** [review]
just a few comments for docs

File: server/resp/src/main/java/org/infinispan/server/resp/logging/AccessData.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+import java.time.temporal.Temporal;
+
+import org.infinispan.security.Security;
+import org.infinispan.server.core.transport.ConnectionMetadata;
+import org.infinispan.server.resp.RespCommand;
+
+import io.netty.channel.ChannelFuture;
+import io.netty.channel.ChannelHandlerContext;
+
+public class AccessData {
```
[the remark this answers is not in the document]
Comment (jabolina) [reply server/resp/src/main/java/org/infinispan/server/resp/logging/AccessData.java:22]: Added some comments.

File: server/resp/src/main/java/org/infinispan/server/resp/logging/Tracker.java (hunk 1 above)
Comment (jabolina) [reply server/resp/src/main/java/org/infinispan/server/resp/logging/Tracker.java:24]: Added some comments.

**Pull request merged**